src/PrestaShopBundle/Resources/views/Admin/Sell/Address/Blocks/required_fields.html.twig

```diff
@@ -0,0 +1,48 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<div class="collapse" id="addressRequiredFieldsContainer">
+  {{ form_start(requiredFieldsForm, {'action': path('admin_addresses_save_required_fields')}) }}
+  <div class="card" >
+    <h3 class="card-header">
+      {{ 'Required Fields'|trans }}
+    </h3>
+    <div class="card-body">
+      <div class="alert alert-info" role="alert">
+        <div class="alert-text">
+          <p>{{ 'Select the fields you would like to be required for this section.'|trans }}</p>
```

Comment:
Do we want to keep this notification? It does not bring any further information apart from repeating it three times:

<img width="1538" alt="Capture d’écran 2019-08-26 à 10 36 11" src="https://user-images.githubusercontent.com/32565890/63677574-91467080-c7ed-11e9-8e51-39f6832d8d74.png">

@colinegin, @marionf, @samuel-pires, what do you think?

Reply:
Maybe we can add an information message / tooltip here, to say that before putting a mandatory field, you need to check if this field is well in the address format in International> Locations> Countries
Like we did here: https://github.com/PrestaShop/PrestaShop/pull/15180#discussion_r316551995
@LouiseBonnard wdyt ?

Reply:
Great idea! What about _Required fields apply to the customer's registration form, you should check the address formats in [1]International > Locations > Countries[/1] before._ localized in _Admin.Orderscustomers.Feature_?